internal/util/sessionutil/session_util.go

```diff
@@ -372,7 +383,6 @@ func (s *Session) registerService() (<-chan *clientv3.LeaseKeepAliveResponse, er
 
 		keepAliveCtx, keepAliveCancel := context.WithCancel(context.Background())
 		s.keepAliveCancel = func() {
```

Comment:
simplify it to : s.keepAliveCancel = keepAlibeCancel ?
you can fix this in next pr.